service/src/main/java/uk/nhs/adaptors/gpc/consumer/filters/SslContextBuilderWrapper.java

```diff
@@ -1,52 +1,36 @@
 package uk.nhs.adaptors.gpc.consumer.filters;
 
 import java.security.KeyStore;
-import java.util.ArrayList;
 import java.util.UUID;
 
 import javax.net.ssl.KeyManagerFactory;
 import javax.net.ssl.TrustManagerFactory;
 
-import org.apache.commons.lang3.StringUtils;
-
 import com.heroku.sdk.EnvKeyStore;
 
 import io.netty.handler.ssl.SslContext;
 import lombok.SneakyThrows;
 import lombok.extern.slf4j.Slf4j;
-import uk.nhs.adaptors.gpc.consumer.gpc.exception.GpConnectException;
+import org.springframework.beans.factory.annotation.Value;
 import uk.nhs.adaptors.gpc.consumer.utils.PemFormatter;
 
+
 @Slf4j
 public class SslContextBuilderWrapper {
+    @Value("${gpc-consumer.gpc.clientKey}")
     private String clientKey;
+    @Value("${gpc-consumer.gpc.clientCert}")
     private String clientCert;
-    private String rootCert;
-    private String subCert;
-
-    public SslContextBuilderWrapper clientKey(String clientKey) {
-        this.clientKey = toPem(clientKey);
-        return this;
-    }
-
-    public SslContextBuilderWrapper clientCert(String clientCert) {
-        this.clientCert = toPem(clientCert);
-        return this;
-    }
-
-    public SslContextBuilderWrapper rootCert(String rootCert) {
-        this.rootCert = toPem(rootCert);
-        return this;
-    }
-
-    public SslContextBuilderWrapper subCert(String subCert) {
-        this.subCert = toPem(subCert);
-        return this;
-    }
+    @Value("${gpc-consumer.gpc.rootCA}")
+    private String rootCA;
+    @Value("${gpc-consumer.gpc.subCA}")
+    private String subCA;
+    @Value("${gpc-consumer.gpc.sslEnabled}")
+    private boolean sslEnabled;
```

Comment:
Interrresting does this get injected successfully? In my head this isn't a property, but is a field on a Configuration class. 🤔

Reply:
I am not 100% confident now that it does, as it appears none of our integration tests cover it and my configration tests only read from the property directly, maybe will have to wait for monday for me to try and add some more tests in for this.